integrations/okta/actions/list-all-users.ts

```diff
@@ -0,0 +1,28 @@
+import type { NangoAction, ProxyConfiguration } from '../../models';
+import { OktaListUsersResponse, OktaUser } from '../types';
+
+export default async function runAction(nango: NangoAction): Promise<OktaListUsersResponse> {
+    const config: ProxyConfiguration = {
+        // https://developer.okta.com/docs/api/openapi/okta-management/management/tag/User/#tag/User/operation/listUsers
+        endpoint: `/api/v1/users`,
```

Comment:
Add in pagination and also filter on `lastUpdated` property to make it an incremental sync https://developer.okta.com/docs/api/openapi/okta-management/management/tag/User/#tag/User/operation/listUsers!in=query&path=filter&t=request

Reply:
done